src/core/shelf.sol

```diff
@@ -94,7 +94,9 @@ contract Shelf is TitleOwned {
 
     // Move the NFT out of the shelf
     function move(uint loan, address registry_, uint nft_, address to) public owner(loan) {
+        require(locks[loan] == 0, "nft-locked");
         NFTLike(registry_).transferFrom(address(this), to, nft_);
+        // TODO: Shouldn't this only be allowed if debt == 0?
```

Comment:
good catch 🥇 

I justed looked it up it happened during the refactoring from `auth`. The method should be internal.